lib/msf/core/exploit/remote/http/manage_engine_adaudit_plus/json_post_data.rb

```diff
@@ -10,7 +10,7 @@ module Msf::Exploit::Remote::HTTP::ManageEngineAdauditPlus::JsonPostData
   # @option options [Integer] :GPO_VERSION The version number of the GPO GUID in use, or a random number from 1 to 9 if one is not supplied.
   # @option options [String] :VER_FILE_NAME The version file name in a format that matches ADAudit Plus's VER_FILE_NAME format.
   # @option options [String] :xmlReport An XML string containing the header to use for the report.
-  # @option options [String] :html_fileName The filename to use for the post request if provided.
+  # @option options [String] :Html_fileName The filename to use for the post request if provided.
```

Comment:
This violates Ruby's camelCase styling so I think the original naming style was appropriate.